File: test/fixtures/matlab/get_with_data.m
%% Web Access using Data Import and Export API
params = {
    'test' '2'
    'limit' '100'
    'w' '4'
};
baseURI = 'http://localhost:28139/synthetics/api/v3/monitors';
uri = [baseURI '?' char(join(join(params, '='), '&'))];
options = weboptions('HeaderFields', {'X-Api-Key' '123456789'});
response = webread(uri, options);

%% HTTP Interface
import matlab.net.*
import matlab.net.http.*

params = {
    'test' '2'
    'limit' '100'
    'w' '4'
};
header = HeaderField('X-Api-Key', '123456789');
uri = URI('http://localhost:28139/synthetics/api/v3/monitors', QueryParameter(params'));
response = RequestMessage('get', header).send(uri.EncodedURI);

File: test/fixtures/ruby/get_with_data.rb
require 'net/http'

uri = URI('http://localhost:28139/synthetics/api/v3/monitors')
params = {
  :test => '2',
  :limit => '100',
  :w => '4',
}
uri.query = URI.encode_www_form(params)

req = Net::HTTP::Get.new(uri)
req['X-Api-Key'] = '123456789'

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end
